corehq/apps/userreports/tasks.py

```diff
@@ -35,7 +35,6 @@ def rebuild_indicators(indicator_config_id):
             adapter.save(doc)
         except DataError as e:
             logging.exception('problem saving document {} to table. {}'.format(doc['_id'], e))
-    adapter.engine.dispose()
```

Comment:
That sounds right but to be honest I'm not 100% sure. In the current code
we're creating a new engine each time the task is run so to me it makes
sense to dispose of it at the end. If we moved to a shared engine instance
then I agree that disposing it doesn't make sense.

Also, by default the engine isn't thread safe (
http://docs.sqlalchemy.org/en/rel_0_9/core/connections.html#using-the-threadlocal-execution-strategy
).

If we do remove it from here then I think this one should also be removed:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/sql/adapter.py#L76-76